### PR TITLE
Pull request as requested

### PR DESCRIPTION
**Pull request by craya1982**
Demo server located in /server folder
Demo app located in /app folder

### CODE REVIEW

**Comment by wprater** [comment]
@craya1982 can you add an `auto-refresh strategy` based off the `expires_at` field?  And we can set the timer to 10s or so, so it's easy to see the refresh happening.  

I would also like to see the a Home view per the mockup so we can see a navigational element to the app.  We will finish the code review after this.  Thank you!

**Comment by craya1982** [comment]
@wprater 
I set the server expiration to 10s as requested.  The app already refreshes based off that field.  When the activity receives successful data from the api service object, it calls through to scheduleTimeout(data.expires_at) and does the appropriate scheduling.

I'll add the home screen later tonight as requested.

**Comment by craya1982** [comment]
@wprater 
Added in fab navigation using fragments along with stub scanner fragment.

File: app/app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    implementation 'com.squareup.retrofit2:retrofit:2.3.0'
+    implementation 'com.squareup.retrofit2:converter-gson:2.3.0'
+    implementation 'io.reactivex.rxjava2:rxjava:2.1.0'
+    implementation 'com.squareup.retrofit2:adapter-rxjava2:2.3.0'
+    implementation 'io.reactivex.rxjava2:rxandroid:2.0.1'
+
+    implementation 'androidx.constraintlayout:constraintlayout:2.0.0-alpha2'
+
+    implementation 'com.google.android.material:material:1.0.0-rc01'
```
[the remark this answers is not in the document]
Comment (craya1982) [reply app/app/build.gradle]: Good catch.  I missed that when I did the AndroidX conversion tool

**Comment by wprater** [comment]
Looking great so far @craya1982! Checking the new build now.

**Comment by wprater** [comment]
Thanks for your submission!

**Pull request closed without merging**